### PR TITLE
fix(llc): separate mute from channel mutes

### PR DESCRIPTION
**Pull request by xsahil03x**
Fixes: #1046

### CODE REVIEW

**Review by ayush221b — APPROVED** [review]

File: packages/stream_chat/lib/src/core/models/channel_mute.dart [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:stream_chat/src/core/models/channel_model.dart';
+import 'package:stream_chat/src/core/models/user.dart';
+
+part 'channel_mute.g.dart';
+
+/// The class that contains the information about a muted user
```
Comment (xsahil03x) [inline packages/stream_chat/lib/src/core/models/channel_mute.dart:7]:
```suggestion
/// The class that contains the information about a muted channel
```

File: packages/stream_chat/lib/src/core/models/channel_mute.dart [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  });
+
+  /// Create a new instance from a json
+  factory ChannelMute.fromJson(Map<String, dynamic> json) =>
+      _$ChannelMuteFromJson(json);
+
+  /// The user that performed the muting action
+  final User user;
+
+  /// The target user
```
Comment (xsahil03x) [inline packages/stream_chat/lib/src/core/models/channel_mute.dart:26]:
```suggestion
  /// The target channel
```

File: packages/stream_chat/lib/src/core/models/channel_mute.dart [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  factory ChannelMute.fromJson(Map<String, dynamic> json) =>
+      _$ChannelMuteFromJson(json);
+
+  /// The user that performed the muting action
+  final User user;
+
+  /// The target channel
+  final ChannelModel channel;
+
+  /// The date in which the use was muted
```
Comment (ayush221b) [inline packages/stream_chat/lib/src/core/models/channel_mute.dart:29]: I think it should be `user`

**Review by ayush221b — CHANGES_REQUESTED** [review]

File: packages/stream_chat/lib/src/core/models/channel_mute.dart (hunk 3 above)
Comment (xsahil03x) [reply packages/stream_chat/lib/src/core/models/channel_mute.dart:29]:
```suggestion
  /// The date in which the channel was muted
```

**Review by ayush221b — APPROVED** [review]

**Pull request merged**